funnel/static/sass/_layout.scss

```diff
@@ -663,3 +663,7 @@ div.markdown-field a.button {
 .rsvp-header {
     padding: 1px 0px 1px 15px
 }
+
+.outstation-speaker {
+    vertical-align: middle;
```

Comment:
You shouldn't need this class. Check what is wrong in the container that icons aren't getting aligned properly.